panels/config/config-entries/ha-config-entries.html

```diff
@@ -13,60 +13,84 @@
   <template>
     <style include='iron-flex ha-style'>
       .content {
-        padding: 8px;
+        padding-bottom: 24px;
       }
-      .entries {
-        margin: 0 -4px;
-      }
-      .entries paper-card {
-        margin: 4px;
+      paper-button {
+        color: var(--primary-color);
+        font-weight: 500;
+        top: 3px;
+        height: 37px;
```

Comment:
We should not control the height of paper-button as that might impact the rendering.